modules/local/spades.nf

```diff
@@ -27,15 +15,15 @@ process SPADES {
     path "${meta.id}_contigs.fasta.gz"                 , emit: contigs_gz
     path "${meta.id}_scaffolds.fasta.gz"               , emit: assembly_gz
     path "${meta.id}_graph.gfa.gz"                     , emit: graph
-    path '*.version.txt'                               , emit: version
+    path "versions.yml"                                , emit: versions
 
     script:
-    def software = getSoftwareName(task.process)
+    def spades_options = params.spades_options ?: ''
```

Comment:
Shouldn't this be passed via `ext.args`? If you are evaluating `spades_options` somewhere 'higher' (i.e. workflow, or subworkflow) this can also be done in `modules.config`

Reply:
You are right, that is not evaluated at any other point, so I moved it so `modules.config`.